src/rust/engine/fs/src/store.rs

```diff
@@ -190,6 +197,11 @@ impl Store {
     )
   }
 
+  ///
+  /// Loads bytes from remote cas. Takes two functions f_local and f_remote. These functions are
```

Comment:
This one is also a little bit of a lie now :) It only backfills if needed

Reply:
lol - modified a bit. ptal when you can